benchmarks/Cargo.toml

```diff
@@ -36,3 +36,6 @@ harness = false
 [[bench]]
 name = "room_bench"
 harness = false
+
+[package.metadata.release]
+release = false
```

Comment:
What does this do? I tried looking for some info but I couldn't find any docs 🫤 .

Reply:
It tells `cargo-release` that it should not consider this crate for a release. It's sparingly documented here: https://github.com/crate-ci/cargo-release/blob/master/docs/reference.md#config-fields.

![image](https://github.com/user-attachments/assets/851cc18d-4ffc-4b01-9e8f-9ebce9e5ef1d)